src/averageAges.js

```diff
@@ -14,12 +14,26 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
+
+function calculateMenAverageAge(people, century = 1) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let peopleArr = [];
+  let sumOfAges = 0;
+
+  century > 1
+    ? peopleArr = people.filter((person) =>
+      Math.ceil(person.died / 100) === century && person.sex === 'm')
+    : peopleArr = people.filter((person) => person.sex === 'm');
+
+  peopleArr.forEach((person) => {
```

Comment:
Replace the forEach loop with a reduce function to calculate the sum of ages. Example: `sumOfAges = peopleArr.reduce((sum, person) => sum + (person.died - person.born), 0);`